test/integration/driver.go

```diff
@@ -83,16 +82,14 @@ func init() {
 		"kube config file or empty for in-cluster")
 	flag.StringVarP(&hub, "hub", "h", "gcr.io/istio-testing",
 		"Docker hub")
-	flag.StringVarP(&tag, "tag", "t", "",
-		"Docker tag (default <username>_YYYMMDD_HHMMSS formatted date)")
+	flag.StringVarP(&tag, "tag", "t", "test",
+		"Docker tag")
```

Comment:
driver.go no longers builds/tags/pushes the docker images. I moved this all back to bin/e2e.sh which can invoke 'bazel run' directly and avoid the issues we were seeing with missing layers/images in Jenkins.